release/Magarena/scripts/Kumano_s_Blessing.groovy

```diff
@@ -0,0 +1,19 @@
+[
+   new DamageIsDealtTrigger(MagicTrigger.REPLACEMENT) {
```

Comment:
same problem as earlier "dealt damage by would die" cards, this implementation ignores the source of the effect once it is put on the creature.

Reply:
removed the card from the request